app/models/concerns/csv_exporter.rb

```diff
@@ -0,0 +1,17 @@
+require 'csv'
+
+module CsvExporter
+  extend ActiveSupport::Concern
+
+  module ClassMethods
+    def to_csv(columns: column_names, csv_options: {})
+      CSV.generate(csv_options) do |csv|
+        csv << columns.map(&:to_s)
+        columns.map!(&:to_sym)
```

Comment:
`column_names` returns a list of strings, I want to be able to pass a list of symbols as well, and I need to make sure they are in the correct format in both places - headers are strings, sent parameters are symbols.